infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -19,6 +19,27 @@ LLVM_DEP_PACKAGES="build-essential make cmake ninja-build git subversion python2
 apt-get install -y $LLVM_DEP_PACKAGES
 
 # Checkout
+CHECKOUT_RETRIES=10
+function checkout_with_retries {
+  REPOSITORY=$1
+  LOCAL_PATH=$2
+  CHECKOUT_FAILED=1
+
+  set -e
+  for i in $(seq 1 $CHECKOUT_RETRIES); do
+    rm -rf $LOCAL_PATH && svn co $REPOSITORY $LOCAL_PATH
+    if [ $? -eq 0 ]; then
+      CHECKOUT_FAILED=0
```

Comment:
You don't need CHECKOUT_FAILED at all, just return here and then if outside loop, it failed.